presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -221,10 +221,20 @@ MySQL Connector Limitations
 The following SQL statements are not yet supported:
 
 * :doc:`/sql/delete`
-* :doc:`/sql/alter-table`
-* :doc:`/sql/create-table` (:doc:`/sql/create-table-as` is supported)
+* :doc:`/sql/create-table` with PRIMARY KEY
 * :doc:`/sql/grant`
 * :doc:`/sql/revoke`
 * :doc:`/sql/show-grants`
 * :doc:`/sql/show-roles`
 * :doc:`/sql/show-role-grants`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/update`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/analyze`
+
+The following supported SQL statements with certain conditions:
```

Comment:
If you're going to keep this format, suggest moving the section of "supported SQL statements" to before the "unsupported SQL statements" section. That way, we can help the reader find what they _can_ do first, before they are told what they _can't_ do.